maintenance/migratestatus/README.md

```diff
@@ -25,6 +25,6 @@ The migratestatus binary is run locally and can be built by running `go build` f
 - `--dest` The destination context. This is the context that is copied **to** and/or the context that replaces the retired context. This flag may only be omitted if retire mode is specified and the old context is being retired without a replacement.
 
 The binary also accepts the following optional parameters:
```

Comment:
Wouldn't it just be easier (and more forwards compatible) to tell people to `go run k8s.io/maintenance/migratestatus -h` to read all this?